CHANGELOG.md

```diff
@@ -1,10 +1,17 @@
 Changes by Version
 ==================
 
-2.9.1 (unreleased)
-------------------
+2.10.0 (2017-11-14)
+-------------------
 
-- nothing yet
+- Support custom tracing headers (#176)
+- Add BaggageRestrictionManager (#178)
+- Add RemoteBaggageRestrictionManager (#182)
```

Comment:
Combine: Add BaggageRestrictionManager (#178) and RemoteBaggageRestrictionManager (#182), it's a single user-facing feature